### PR TITLE
CC: support PPM on pin8, fix oneshot

### PR DESCRIPTION
**Pull request by mlyle**
Fixes issue #1753 .

Also a minor packaging fix from target names getting desynched in makefiles, fixes windows packaging.

~~Test binaries at http://jar.lyle.org/~mlyle/TauLabs-20150807-efb4c19b-install.exe~~

Revised test binaries at http://jar.lyle.org/~mlyle/TauLabs-20150808-e2a092cd-install.exe

### CODE REVIEW

**Comment by mlyle** [comment]
Two known issues:

1. Doesn't arm ESCs on boot when in PIN8 settings--- we are obviously relying on some side effect from PPM configuration.  So this needs work/analysis.
2. Wiring diagram for PPM is bad in GCS.

**Comment by SystemsGuy** [comment]
Works here on CC3D with PPM+8.  Connection diagram needs to be updated.  Flys + autotunes fine with FastPWM, moving the F/C to a Oneshot system now.

Oneshot arms and appears to work on the bench.

Memory Warning when Autotune is enabled.  It's probably worth updating the text description to reflect that as "normal".

**Comment by soend** [comment]
Tested this on bench with CC3D and D4R-II connected to PIN 8.
In output configuration Update rate: SyncPWM and Resolution: 12Mhz.

Works fine.

File: flight/PiOS/STM32F10x/pios_tim.c [hunk 1]
```diff
@@ -137,7 +137,7 @@ int32_t PIOS_TIM_InitChannels(uintptr_t * tim_id, const struct pios_tim_channel
 			RCC_APB2PeriphClockCmd(RCC_APB2Periph_GPIOB, ENABLE);
 			break;
 		case (uint32_t) GPIOC:
-			RCC_APB2PeriphClockCmd(RCC_APB2Periph_GPIOB, ENABLE);
+			RCC_APB2PeriphClockCmd(RCC_APB2Periph_GPIOC, ENABLE);
```
Comment (peabody124) [inline flight/PiOS/STM32F10x/pios_tim.c]: good catch

File: flight/targets/coptercontrol/fw/pios_board.c [hunk 2]
```diff
@@ -856,11 +856,13 @@ void PIOS_Board_Init(void) {
 #endif	/* PIOS_INCLUDE_PWM */
 		break;
 	case HWCOPTERCONTROL_RCVRPORT_PPM:
```
Comment (peabody124) [inline flight/targets/coptercontrol/fw/pios_board.c]: want to call this PPMONPIN1 for consistency? maybe not worth breaking .uav files.

File: package/Makefile [hunk 3]
```diff
@@ -45,7 +45,7 @@ CLEAN_FLIGHT := YES
 endif
 
 # Set up targets
-ALL_BOARDS        := coptercontrol pipxtreme revomini sparky sparky2 sparkybgc quanton colibri flyingf3 flyingf4 aq32
+ALL_BOARDS        := coptercontrol pipxtreme revomini sparky sparky2 sparkybgc quanton colibri flyingf3 flyingf4 aq32 naze32
```
Comment (peabody124) [inline package/Makefile]: not good to mix unrelated things

**Comment by peabody124** [comment]
LGTM if flight tested

File: package/Makefile (hunk 3 above)
Comment (mlyle) [inline package/Makefile]: Yah.. But I needed it anyway on the branch to get windows packaged builds.

Just as an aside packaging is broken everywhere-- on OSX by the QT5.5 change, by this.. That's something I'm going to look at systemically in a week or two.

**Comment by mlyle** [comment]
Merging based on comment.

File: flight/targets/coptercontrol/fw/pios_board.c (hunk 2 above)
Comment (mlyle) [inline flight/targets/coptercontrol/fw/pios_board.c]: Yah I didn't want to break existing stuff.  It's not the default PPM anymore which is a little weird, but... meh.

**Pull request merged**